controllers/configuration_controller.go

```diff
@@ -710,6 +730,10 @@ func (meta *TFConfigurationMeta) assembleTerraformJob(executionType TerraformExe
 			Name:      BackendVolumeName,
 			MountPath: BackendVolumeMountPath,
 		},
+		{
+			Name:      GitAuthConfigVolumeName,
+			MountPath: GitAuthConfigVolumeMountPath,
+		},
```

Comment:
Not all git repo need authentication, please modify all logic that always do it.

Reply:
Done